### DISCUSSION TITLE
IMSC: is it still relevant or should it be removed?

### DISCUSSION THREAD

**Discussion by estelle** [mdn/mdn-community#483] [Content]
### MDN URL

https://developer.mozilla.org/en-US/docs/Related/IMSC

### What specific section or headline is this issue about?

Is this page useful?

### What information was incorrect, unhelpful, or incomplete?

IMSC is TTML Profiles for Internet Media Subtitles and Captions (IMSC). 
The spec hasn't been updated in 4 years:https://github.com/w3c/imsc
It requires a polyfill, i think, https://github.com/sandflow/imscJS. 
Either we should update the browser support, if any (and i don't think there is any), or we should remove it.

Text Tracks seems to be going in a different direction, but I may be thinking one thing is two different things. see https://github.com/whatwg/html/pull/9771 for the path they are on.

### What did you expect to see?

relevant, supported related content.

### Do you have any supporting links, references, or citations?

Note that we have an entire section of the site called "related web technologies" and the only content is IMSC. When we integrate it into real content or delete it, we can delete https://developer.mozilla.org/en-US/docs/Related.

**Comment by bsmth** [comment]
Hi @estelle - I think it makes sense to move this over to a Discussion, what do you think?
